src/components/screens/signMultiSignTransaction/importData/importData.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+import Box from '../../../toolbox/box';
+import BoxContent from '../../../toolbox/box/content';
+import BoxFooter from '../../../toolbox/box/footer';
+import { PrimaryButton } from '../../../toolbox/buttons';
+import ProgressBar from '../progressBar';
+import styles from './styles.css';
+
+const ImportData = ({ t, nextStep }) => {
+  const [jsonInput, setJsonInput] = useState(undefined);
+  const reader = new FileReader();
+
+  reader.onload = ({ target }) => {
```

Comment:
I think we should move `const reader = new FileReader();` to out of the component and add listeners to the reader instance in `useEffect`